src/platform/site-wide/mega-menu/components/MegaMenu.jsx

```diff
@@ -87,6 +86,7 @@ export default class MegaMenu extends React.Component {
   handleDocumentClick = event => {
     if (this.props.currentDropdown && !this.menuRef.contains(event.target)) {
       this.props.toggleDropDown('');
+      this.props.updateCurrentSection('');
```

Comment:
I don't think call to blank out the current section is strictly necessary here since it will be blanked out when the menu is clicked (line 171). That said, it might be more clear and safe to have it here. I don't have a strong feeling.

Reply:
Yes, this makes sense. I think what I poorly attempted to say is that it doesn't really matter if the previously selected section is deselected when the menu is hidden. It will be deselected when it is clicked to become visible again. As I mentioned, though, this is likely not worth worrying too much about. It's arguable a good safety net, though perhaps at a very small cost.